var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -250,6 +250,6 @@ def write_rpath_specs(self):
             for line in lines:
                 out.write(line + '\n')
                 if line.startswith('*link:'):
-                    out.write(r'-rpath {0}:{1} \n'.format(
```

Comment:
the original line https://github.com/LLNL/spack/pull/4144/files#diff-38bf8d6101b9bc88270ac094f38565e1L186 was
```
out.write("-rpath %s/lib:%s/lib64 \\\n" %
                   (self.prefix, self.prefix))
```
so we are dropping that `\\\n` here. 

Not that i know the difference in Python between `\n` and `\\\n` :smile: